client/components/pages/files/FilesTable.vue

```diff
@@ -3,23 +3,34 @@ import type {
   DataTablePageEvent,
   DataTableRowContextMenuEvent,
   DataTableRowDoubleClickEvent,
-  DataTableSortEvent,
+  DataTableSortMeta,
 } from 'primevue/datatable'
 import type { PContextMenu } from '#components'
 
 const DEFAULT_PAGE = 0
-const DEFAULT_PAGE_SIZE = 10
+const DEFAULT_PAGE_SIZE = 25
 
 const { locale, t, d } = useI18n()
+const filters = useFilesFilters()
+const { format } = useHumanFileSize()
 
 const selectedRows = defineModel<Array<FileRepresentationDTO>>('selectedRows', { default: () => [] })
 const selectedRow = ref<Nullable<FileRepresentationDTO>>(null)
+const sort = ref<DataTableSortMeta[]>([{ field: 'createdAt', order: -1 }])
+const query = defineModel<string>('query', { default: () => '' })
+const debouncedQuery = useDebounce(query, 500)
 const page = ref(DEFAULT_PAGE)
 const pageSize = ref(DEFAULT_PAGE_SIZE)
-const sortDefinition = ref<SorterDefinitionDTO>([])
-
-const pageDefinition = computed<PageDefinitionDTO>(() => ({ sorter: sortDefinition.value }))
+const sortDefinition = computed(() => sort.value.map<SortObject>(({ field, order }) => ({
+  field,
+  direction: order === 1 ? 'ASC' : 'DESC',
+})))
+const filterDefinition = computed(() => filtersToDefinition(filters, debouncedQuery.value))
 
+const pageDefinition = computed<PageDefinitionDTO>(() => ({
```

Comment:
I think we may not need this computed value.

You probably can just pass the body as 
```js
body: {
    sorter: sortDefinition,
    filters: filterDefinition
}
```

Reply:
Unfortunately, this doesn't seem to work...

Reply:
I thought this could work like the query parameter but eh, it's alright

Reply:
Yeah, not a big deal if you ask me